### PR TITLE
Selected Item

### PR DESCRIPTION
**Pull request by MCurran16**
Fix Issue #19

### CODE REVIEW

File: assets/js/vm/app.js.coffee [hunk 1]
```diff
@@ -11,8 +11,14 @@ do ->
             subtitle = @subtitle()
             return !(subtitle and subtitle.length)
 
+
+         ###-----------------------------------
+                  Header Menu Items
+         -----------------------------------###
          @selectedMenuItem = ko.observable("home")
 
+         @isSignedIn = ko.observable(false)
```
Comment (hunterbrowning) [inline assets/js/vm/app.js.coffee]: We will change this when we do Auth but this is fine to demo the functionality for now.

File: assets/js/vm/app.js.coffee [hunk 2]
```diff
@@ -22,6 +28,10 @@ do ->
          assertArgs(arguments, PageContentViewModel)
          @setChild("main", viewModel)
 
+      setSelectedItem: (menuItem) ->
```
Comment (hunterbrowning) [inline assets/js/vm/app.js.coffee]: lets call this `setSelectedNavItem`

File: assets/js/app-layout.jst.jade [hunk 3]
```diff
@@ -38,8 +38,17 @@ header
 							a(href="/about")
 								span.header__menu-item-text(data-bind="click: function(){selectItem('about')}, \
 										css: {'menu-item--active': selectedMenuItem() == 'about'}") about
+						<!-- ko if: isSignedIn() -->
+						li.header__menu-item.menu-item--profile(data-bind="css: {'selected-menu-border': selectedMenuItem() == 'profile'}, \
+								style: {margin: '0 30px 0 0'}")
```
Comment (hunterbrowning) [inline assets/js/app-layout.jst.jade]: Why are these styles here instead of in a style sheet?

File: assets/js/vm/app.js.coffee (hunk 2 above)
Comment (MCurran16) [inline assets/js/vm/app.js.coffee]: fixed

File: assets/js/app-layout.jst.jade (hunk 3 above)
Comment (MCurran16) [inline assets/js/app-layout.jst.jade]: oops, i thought i took them out. fixed

File: assets/js/vm/app.js.coffee (hunk 1 above)
Comment (MCurran16) [inline assets/js/vm/app.js.coffee]: Yes, I am just using it temporarily for functionality.

**Comment by hunterbrowning** [comment]
LGTM

**Pull request merged**